src/scalems/execution.py

```diff
@@ -264,24 +333,16 @@ async def __aexit__(self, exc_type, exc_val, exc_tb):  # noqa: C901
                 else:
                     self._exception = e
             finally:
-                # Should we do any other clean-up here?
```

Comment:
@andre-merzky suggests that normative usage should be to explicitly shut down each component. I believe this means
1. `pilot.cancel()`
2. `task_manager.close()`
3. `pilot_manager.close()`
4. `session.close()`